service/version_bundles.go

```diff
@@ -0,0 +1,38 @@
+package service
+
+import (
+	"time"
+
+	"github.com/giantswarm/versionbundle"
+)
+
+func NewVersionBundles() []versionbundle.Bundle {
+	return []versionbundle.Bundle{
+		{
+			Changelogs: []versionbundle.Changelog{
+				{
+					Component:   "node-operator",
+					Description: "Introduce the first version of the node-operator.",
+					Kind:        versionbundle.KindAdded,
+				},
+			},
+			Components: []versionbundle.Component{
+				{
+					Name:    "node-operator",
+					Version: "0.1.0",
+				},
```

Comment:
This is a bit weird. I am not sure what the component for an operator is which does not provide anything but itself. Since we do not collect the version bundles here for release aggregation I think this is fine. It is just to have the versioning and visibility via version bundles.

Reply:
This may mean we may allow zero components. This makes sense.

Reply:
Maybe. Uncertain.